src/simtools/visualization/plot_incident_angles.py

```diff
@@ -0,0 +1,419 @@
+#!/usr/bin/python3
+"""Plot incident angle histograms for focal, primary, and secondary mirrors.
+
+Also plots the primary-mirror hit radius if available.
+"""
+
+import logging
+from pathlib import Path
+
+import astropy.units as u
+import matplotlib.pyplot as plt
+import numpy as np
+
+__all__ = ["plot_incident_angles"]
+
+Y_AXIS_BIN_COUNT_LABEL = "Count / Bin"
```

Comment:
As mentioned earlier, distributions should be normalized to integral one.

Reply:
fixed

Reply:
Do you normalize in plot or calculate incident angle? It should be done in calculate incidient angle, as we want to write out there as well the results to the simulation model.